kolibri/plugins/management/assets/src/views/facilities-config-page/index.vue

```diff
@@ -128,6 +129,7 @@
       learnerCanEditPassword: 'Allow users to change their password when signed in',
       learnerCanEditUsername: 'Allow users to edit their username',
       learnerCanSignUp: 'Allow users to sign-up on this device',
+      learnerCanLoginWithNoPassword: 'Allow learners to sign in with no password',
```

Comment:
Unfortunately, I can't change this, as it directly maps to the snake case name of the field on the dataset model.